src/polytopes/hexahedron.jl

```diff
@@ -28,15 +28,14 @@ function (h::Hexahedron)(u, v, w)
 end
 
 function measure(h::Hexahedron)
-  A1, A2, A4, A3,
+  A1, A2, A4, A3, 
```

Comment:
Please eliminate this whitespace change.

Reply:
haha,ok👌